package lv.akurss.lesson5.lecture._1_exception_catch;

public class ArrayIndexOutOfBoundsExceptionExample {

	public static void main(String[] args) {
		int[] a = {1, 2, 3};
		System.out.println(a[4]);
		System.out.println("Finish");
	}
	
}
